API.md

```diff
@@ -131,15 +132,25 @@ requests. May be cached offline.
 
 ##### `VolunteerInfo` interface
 
+| Property          | Type      | Description |
+| :---              | :---      | :--- |
+| `userToken`       | `string`  | The token that identifies this volunteer. Should be pseudo-anonymous. |
+| `groupToken`      | `string`  | The token that identifies the group this volunteer is part of. |
+| `name`            | `string`  | The full name of this volunteer. |
+| `avatar`          | `string?` | URL to the avatar that's to be displayed for this volunteer. |
+| `title`           | `string`  | Title to be displayed for this volunteer. |
+| `accessCode`      | `string?` | Access code of this volunteer. Should be restricted to admins. |
+| `telephone`       | `string?` | Telephone number of this volunteer. Should be restricted. |
+| `lastLogbookRead` | `number?` | Time, in seconds since the UNIX epoch, of the last logbook entry the user has read. |
+
+##### `LogbookEntry` interface
+
 | Property         | Type      | Description |
 | :---             | :---      | :--- |
-| `userToken`      | `string`  | The token that identifies this volunteer. Should be pseudo-anonymous. |
-| `groupToken`     | `string`  | The token that identifies the group this volunteer is part of. |
-| `name`           | `string`  | The full name of this volunteer. |
-| `avatar`         | `string?` | URL to the avatar that's to be displayed for this volunteer. |
-| `title`          | `string`  | Title to be displayed for this volunteer. |
-| `accessCode`     | `string?` | Access code of this volunteer. Should be restricted to admins. |
-| `telephone`      | `string?` | Telephone number of this volunteer. Should be restricted. |
+| `entryTime`      | `number`  | Time, in seconds since the UNIX epoch, at which the entry was recorded. |
+| `message`        | `strings` | The message for a logbook entry. |
+| `seniorOnly`     | `boolean` | Whether an entry is only visible to seniors. (Note that the server should filter out entries the user doesn't have access to.) |
```

Comment:
nit: maybe name this `confidential`? The server could decide to expose (or hide) log entries for any reason, it's just that we're choosing to do it based on volunteer seniority.